README.md

```diff
@@ -99,6 +98,45 @@ tuner.run()
 
 The above example runs ASHA with 4 asynchronous workers on a local machine.
 
+## Supported HPO methods
+
+The following hyperparameter optimization (HPO) methods are available in Syne Tune:
+
+Method | Reference | Searcher | Asynchronous? | Multi-fidelity? | Transfer? 
+:--- | :---: | :---: | :---: | :---: | :---: 
+Grid Search | Someone (sometime) | deterministic | no | no | no 
+Random Search | Bergstra, et al. (2011) | random | yes | no | no 
+Bayesian Optimization | Snoek, et al. (2012) | model-based | yes | no | no 
+BORE | Tiao, et al. (2021) | model-based | yes | no | no 
+MedianStoppingRule | Golovin, et al. (2017) | any | yes | yes | no 
+SyncHyperband | Li, et al. (2018) | random | no | yes | no 
+SyncBOHB | Falkner, et al. (2018) | model-based | no | yes | no 
+SyncMOBSTER | Klein, et al. (2020) | model-based | no | yes | no 
+ASHA | Li, et al. (2019) | random | yes | yes | no 
+BOHB | Falkner, et al. (2018) | model-based | yes | yes | no 
+MOBSTER | Klein, et al. (2020) | model-based | yes | yes | no 
+DEHB | Awad, et al. (2021) | random | no | yes | no 
+HyperTune | Li, et al. (2022) | model-based | yes | yes | no 
+PASHA | Bohdal, et al. (2022)| random | yes | yes | no 
+REA | Real, et al. (2019) | model-based | yes | no | no 
+PBT | Jaderberg, et al. (2017) | model-based | no | yes | no 
+ZeroShotTransfer | Wistuba, et al. (2015) | deterministic | yes | no | yes 
```

Comment:
Perhaps we could use just ZeroShotTransfer given that otherwise all transfer methods could be suffixed with "Transfer".

Reply:
We also have multi-fidelity BORE, please ask Aaron whether this is functional, but it is in the code.

Reply:
I'd say "GP" instead of "model-based", or "Gaussian process"

Reply:
We also have what I'd right now would call BO-KDE or so, namely this is like BOHB, but not multi-fidelity.
Or we can call it TPE-KDE.

Reply:
"model-based" is too broad, use "GP" and "KDE".

One nice idea would be to use the name you pass to searcher in FIFOScheduler, which would be "bayesopt" for GP, and "kde" for BOHB and TPE-KDE

Reply:
PBT is not model-based, it is random like RS. It just does clever scheduling.
We could call it "EA" (for evol algo), in which case DEHB would also be "EA"

Reply:
> I'd say "GP" instead of "model-based", or "Gaussian process"

Some model-based methods are not GP-based. I first used the finer denomination, but then we end up with too many IMHO.

Reply:
> Perhaps we could use just ZeroShotTransfer given that otherwise all transfer methods could be suffixed with "Transfer".

I am not sure I follow. Are you suggesting to drop Transfer?

Reply:
> PBT is not model-based, it is random like RS. It just does clever scheduling. We could call it "EA" (for evol algo), in which case DEHB would also be "EA"

I defined model-based as sampling non-uniforrmly at random so in that sense PBT falls in that category I believe. Using EA would be more accurate of course. Happy to change DEHB, PBT and REA to EA.